CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Added a validation to **validate**, checking that each script, integration and playbook have a README file. This validation only runs when the command is called with either the `-i` or the `-g` flag.
 * Fixed a regression issue with **doc-review**, where  the `-g` flag did not work
 * Improved the detection of errors in **doc-review** command.
+* When **validate** is validating *pack unique files* it will check for README existence only for packs that contain playbooks or writen by partners. 
```

Comment:
```suggestion
* Fixed an issue in **validate**, where README existence was only checked in packs that contain playbooks, or written by partners. 
```

Reply:
That is not really what i have done. the validation was once triggered for all packs containing use cases or written by partners. 
now it validates only for packs how contain playbooks or were written by partners. 
also made a better more clear error message. 
@dorschw

Reply:
got it. plesae push the updated message.